javascript/src/stories/components/SearchedTermsTable/SearchedTermsTable.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
```

Comment:
This entire file does not look related to the feature you are trying to implement, which I'm guessing is this one: https://github.com/ucsb-cs156-w21/proj-mapache-search/issues/203, Again, same comment